PertDB/dbo/StoredProcedures/DeleteDependency.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE PROCEDURE [dbo].[DeleteDependency]
+	@taskId int
+AS
+BEGIN
+	DELETE FROM [dbo].Dependency
+	WHERE RootId = @taskId OR DependentId = @taskId
```

Comment:
This is deleting all incoming and outgoing dependencies? This is already handled in the DeleteTask trigger.

Reply:
for potential cases when we aren't necessarily deleting a task but want to remove all dependencies associated with it